src/Agent.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -297,6 +298,111 @@ public AgentSettings LoadSettings()
+                try
+                {
+                    serviceControlManager.UnconfigureService();
+                    _term.WriteLine(StringUtil.Loc("Success"));
+                }
+                catch (Exception ex)
+                {
+                    Trace.Error(ex);
+                    _term.WriteLine(StringUtil.Loc("Failed"));
+                    throw;
+                }
+            }
+            else
+            {
+                _term.WriteLine(StringUtil.Loc("NotFound"));
+            }
+
+            //delete agent from the server
+            _term.Write(StringUtil.Loc("UnregisteringAgent"));
+            bool isConfigured = _store.IsConfigured();
+            bool hasCredentials = _store.HasCredentials();
+            if (isConfigured && hasCredentials)
+            {
+                try
+                {
+                    AgentSettings settings = _store.GetSettings();
+                    var credentialManager = HostContext.GetService<ICredentialManager>();
+                    VssCredentials creds = credentialManager.LoadCredentials();
+                    Uri uri = new Uri(settings.ServerUrl);
+                    VssConnection conn = ApiUtil.CreateConnection(uri, creds);
+                    var agentSvr = HostContext.GetService<IAgentServer>();
+                    await agentSvr.ConnectAsync(conn);
+                    await agentSvr.DeleteAgentAsync(settings.PoolId, settings.AgentId);
+                    _term.WriteLine(StringUtil.Loc("Success"));
+                }
+                catch (Exception ex)
+                {
+                    Trace.Error(ex);
+                    bool agentNotFound = ex is TaskAgentPoolNotFoundException || ex is TaskAgentNotFoundException;
+                    if (agentNotFound)
+                    {
+                        _term.WriteLine(StringUtil.Loc("NotFound"));
+                    }
+                    else
+                    {
+                        _term.WriteLine(StringUtil.Loc("Failed"));
+                        throw;
+                    }
+                }
+            }
+            else
+            {
+                _term.WriteLine(StringUtil.Loc("MissingConfig"));
+            }
+
+            //delete credential config file
+            _term.Write(StringUtil.Loc("DeletingCredentials"));
+            if (hasCredentials)
+            {
+                try
+                {
+                    _store.DeleteCredential();
+                    _term.WriteLine(StringUtil.Loc("Success"));
+                }
+                catch (Exception ex)
+                {
+                    Trace.Error(ex);
+                    _term.WriteLine(StringUtil.Loc("Failed"));
+                    throw;
+                }
+            }
+            else
+            {
+                _term.WriteLine(StringUtil.Loc("NotFound"));
+            }
+
+            //delete settings config file
+            _term.Write(StringUtil.Loc("DeletingSettings"));
```

Comment:
OK. In this case I'll print the failed, success and not found messages on new lines. It will be like
WriteLine("{currentAction}") -> when the action starts
WriteLine("Failed:  {currentAction}") -> if action fails
WriteLine("Succeeded:  {currentAction}") -> on success
WriteLine("Does not exist.  Skipping {currentAction}") -> if missing config file, service or agent registration